src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,86 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+export const PeopleTable = ({ people }: { people: Person[] }) => {
+  const { slug: urlSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            name,
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+            slug,
+          } = person;
+
+          const findMother = people.find(p => p.name === motherName);
+          const findFather = people.find(p => p.name === fatherName);
```

Comment:
looks like you can create separated function for `.find()` and reuse (not critical)